packages/admin-app/src/components/ElectionSetupWizard/ElectionSetupWizard.tsx

```diff
@@ -29,21 +31,30 @@ export enum ElectionSetupStep {
  * Wizard to setup the election
  */
 export const ElectionSetupWizard: React.FC = () => {
-    const [step, setStep] = useState(ElectionSetupStep.Instructions);
-    const { nextStep } = createEnumStepper(ElectionSetupStep);
-    const next = () => setStep(nextStep(step));
+    const [submitElectionRequest, setSubmitElectionRequest] = useState({} as SubmitElectionRequest);
+    const [step, setStep] = useState(ElectionSetupStep.BasicInfo);
+    const { nextStep: getNextStep } = createEnumStepper(ElectionSetupStep);
+    const navigate = useNavigate();
+    const next = (newSubmitElectionRequest: SubmitElectionRequest) => {
+        setSubmitElectionRequest(newSubmitElectionRequest);
+        const nextStep = getNextStep(step);
+        setStep(nextStep);
+    };
 
     const service = ApiClientFactory.getGuardianApiClient();
     return (
         <Box height="100%">
-            <WizardStep active={step === ElectionSetupStep.Instructions}>
-                <SetupInstructionsStep onNext={next} />
+            <WizardStep active={step === ElectionSetupStep.BasicInfo}>
+                <BasicInfoStep onNext={next} />
             </WizardStep>
             <WizardStep active={step === ElectionSetupStep.JointKeySelect}>
-                <JointKeySelectStep onNext={next} />
+                <JointKeySelectStep submitElectionRequest={submitElectionRequest} onNext={next} />
             </WizardStep>
             <WizardStep active={step === ElectionSetupStep.JointKeyRetrieved}>
-                <JointKeyRetrievedStep onNext={next} />
+                <JointKeyRetrievedStep
+                    submitElectionRequest={submitElectionRequest}
```

Comment:
This component isn't a good example of the pattern I'm going for because that step doesn't make changes to SubmitElectionRequest yet (it will in my next PR).  Check out JointKeySelectStep for the pattern I'm going for, and see if you agree with the pattern.  Sorry, I usually self-review to explain what I'm doing, but forgot to for this PR.

The idea I had was that each step will take the SubmitElectionRequest from the previous step and then when the user clicks next step use the javascript spread operator to modify the prior SubmitElectionRequest with its new values but without mutating state.  I believe this immutable pattern is idiomatic for React.  

Here's what it looks like in JointKeySelectStep:

```
const onNextClick = () => {
    const newSubmitElectionRequest = {
        ...submitElectionRequest,
        key_name: keyCeremony?.key_name,
    } as SubmitElectionRequest;
    onNext(newSubmitElectionRequest);
};
```

I guess I could move the spread operator into the ElectionSetupWizard though if you like and still keep the pattern.  What do you think?